src/igroupadmin.cpp

```diff
@@ -466,14 +466,25 @@ void usageMain() {
         "certain administrative functions.  It can also be used by regular",
         "users to list groups ('lg') and by the admin for all operations.",
         " ",
+        "After making a new group they will themselves manage, a groupadmin's
+        "first action should be to add their own username to the group.",
+        "They will from that point forward be able to add users to, and remove",
+        "them from, the group.  Those operations permit, as well, the transfer",
+        "of responsibility for the group to another groupadmin.",
+        " ",
+        "Currently, a groupadmin may not remove groups or users from the zone."
+        " ",
         "A blank execute line invokes the interactive mode, where it",
         "prompts and executes commands until 'quit' or 'q' is entered.",
         "Single or double quotes can be used to enter items with blanks.",
         "Commands are:",
         " lg [name] (list group info (user member list))",
         " mkuser Name Password (make a user and set the initial password)",
         " atg groupName userName[#Zone] (add to group - add a user to a group)",
+        "   * NOTE: atg is allowed for a groupadmin only if they belong to the group given,",
+        "     and it is currently empty (no members)."
```

Comment:
Is this true? Can't the groupadmin add users to the group if they are a member of the group? Or is it only when they are the *first* member in a group?

Please also consider moving this text to the `atgMsgs` which gives more details about the `atg` subcommand. Users can view this text with `igroupadmin h atg` and the text will appear on this page in the next version (I've linked the 4.3.0 page for demonstration purposes only): https://docs.irods.org/4.3.0/icommands/groupadmin/

This has been the precedent for all the iCommands which use subcommands (`iadmin`, `igroupadmin`, `imeta`, `iticket`). If we want to keep this text front and center, that's fine, just wanted to make mention of this.

Missing a comma at the end of this line.

Reply:
Sorry, meant to say:

Can't the groupadmin add users to the group if they are a member of the group and it's not empty as well?
```bash
# as labmanager1
$ igroupadmin mkgroup lab1
$ igroupadmin atg lab1 labmanager1
$ igroupadmin atg lab1 alice
$ igroupadmin atg lab1 bobby
```
That should work (I thought) and `atg` succeeds when the group is not empty. Maybe I'm just reading the sentence wrong. :)

Reply:
> Is this true? Can't the groupadmin add users to the group if they are a member of the group? Or is it only when they are the _first_ member in a group?
> 
I think youre right;  will revisit

Reply:
I don't know about moving the NOTE annotated stuff to the subcommand help.  To me, that makes it more obscure. It's the reason i wasn't familiar with the special kinks of igroupadmin in the first place. Someone has to know there's stuff to dig for, in the output of "igroupadmin h atg" , ie. more than what one would find in "man igroupadmin",

Reply:
Understood. I defer to others on this matter, then.

Reply:
Me too tbh.  Could do with input from @trel and @korydraughn .  Do we take a dual approach, then ? Teach the philosophy in the main text and give examples within each subcommand text ?

Reply:
I'd be happy to get rid of the subcommand help text - seems like we can fit whatever we need at the top level... ?

Reply:
I'm not a fan of how a person views the help text for a subcommand, but I feel we should move the note about constraints for `atg` and `rfg` into the subcommand help text.

We can include a line in the top level help text that instructs the user to view the subcommand help text of those operations.

For example:
```
Usage: igroupadmin [-vV] [command]
Usage: igroupadmin h [command]

<snip>

NOTE: atg and rfg have additional usage requirements. See the help text
of these subcommands for details (e.g. igroupadmin h atg).

Commands are:
 <snip>
 atg groupName ...
 rfg groupName ...
```

Moving all of the subcommand help text into the top level would work, but it feels incorrect to print subcommand help text at that level.

Willing to be persuaded otherwise.

Reply:
That seems fine with me.